src/nethogs.cpp

```diff
@@ -38,6 +38,14 @@
 #include <netinet/tcp.h>
 #include <netinet/udp.h>
 
+#if NEEDROOT == 0
+#include <linux/limits.h>
```

Comment:
Wouldn't it be neat to scope these imports as narrowly as possible, so move them to main.cpp?